bundles/org.openhab.core.model.yaml/src/main/java/org/openhab/core/model/yaml/internal/rules/YamlModuleDTO.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.model.yaml.internal.rules;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.openhab.core.automation.Module;
+import org.openhab.core.model.yaml.internal.util.YamlElementUtils;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+/**
+ * The {@link YamlModuleDTO} is a data transfer object used to serialize a module in a YAML configuration file.
+ *
+ * @author Ravi Nadahar - Initial contribution
+ */
+public class YamlModuleDTO {
+
+    public String id;
+    public String label;
+    public String description;
+    @JsonAlias({ "configuration" })
```

Comment:
What does it mean ? Does it mean the user can use either "configuration" or "config" in its syntax ?

It was already discussed for things and items that we should use the short name, that is "config".

Reply:
That is a good idea. I would suggest to extend things/items the same way (in a separate PR).